pages/components/Header.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import styles from './Header.module.css';
+
+const Header = () => {
+  return (
+    <header className={styles.header}>
+        <nav>
+            <a href='/' className={styles.nav_item}>Home</a>
+            <a  href='/courses' className={styles.nav_item}>Courses</a>
```

Comment:
To prevent errors when clicking the link, use **"#"** instead of **/** or a named link for both of the links.

Reply:
Links changed to #.